### ISSUE TITLE
PageAction invisible in the RequestPage

### ISSUE THREAD

**Issue by jthoerner-tso** [microsoft/AL#6744]
Hello

If you add a PageAction in the RequestPage of a report, this PageAction is invisible in the client, no matter in which area this PageAction is located. 

It looks like this behaviour is intentional, because under C/AL you could also add PageActions and they were invisible in the client. 
However, this is not documented, so I still have to ask whether this is intentional, or perhaps a bug?

If this is indeed intended, then a warning would certainly be helpful.

``` AL
report 52201 "TestReportREG"
{
    Caption = 'TestReport';
    UsageCategory = Lists;
    ApplicationArea = All;

    requestpage
    {
        actions
        {
            area(Processing)
            {
                action(Test)
                {
                    Image = New;
                    Promoted = true;
                    PromotedCategory = Process;
                    PromotedOnly = true;
                    ApplicationArea = All;
                    trigger OnAction()
                    begin
                        Message('Test');
                    end;
                }
            }
        }
    }
}

```

**Comment by thloke** [comment]
You are right that this is intentional, request pages (and their extensions in report extensions) do not have support for actions. We can certainly add a warning.

**Comment by SBalslev** [comment]
Hi! Thank you for reporting this behavior with PageActions in report RequestPages.

This appears to be a runtime/client-side behavior rather than a compiler issue. The compiler correctly processes the AL code, but the Business Central client controls which actions are rendered on RequestPages.

We're closing this to keep the backlog focused on compiler and developer tooling issues. If you believe this should be changed, please consider posting it on https://aka.ms/BCIdeas.

Thank you for being part of the Business Central developer community!
